client-common/src/tendermint/client.rs

```diff
@@ -37,7 +37,13 @@ pub trait Client: Send + Sync + Clone {
     fn broadcast_transaction(&self, transaction: &[u8]) -> Result<BroadcastTxResponse>;
 
     /// Makes `abci_query` call to tendermint
-    fn query(&self, path: &str, data: &[u8]) -> Result<AbciQuery>;
+    fn query(
+        &self,
+        path: &str,
+        data: &[u8],
+        height: Option<Height>,
+        prove: bool,
```

Comment:
some comments for prove?

Reply:
can prove be optional?

Reply:
Added some comments now.
Prove is to say include proof or not, can pass `false` by default.